### ISSUE TITLE
Alternating training (cell weights/network weights)

### ISSUE THREAD

**Issue by PhilippHa3** [PhilippHa3/pytorch-CycleGAN-and-pix2pix#4]
add the possibility to alternate between training the cell weights and the network weights as it is done in AutoDeepLab.
Needs new optimizer and new learning rate.
